storage/s3_sdk/list_gcs_buckets_test.go

```diff
@@ -17,29 +17,78 @@ package s3sdk
 
 import (
 	"bytes"
+	"context"
+	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
 	"strings"
 	"testing"
+
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
 )
 
+var serviceAccountEmail = os.Getenv("GOLANG_SAMPLES_SERVICE_ACCOUNT_EMAIL")
+var storageClient *storage.Client
+
 func TestMain(m *testing.M) {
 	log.SetOutput(ioutil.Discard)
 	s := m.Run()
 	log.SetOutput(os.Stderr)
+
+	ctx := context.Background()
+	storageClient, _ = storage.NewClient(ctx)
```

Comment:
Check the error. Also, s/storageClient/client/ -- don't need to disambiguate clients here since we are only using storage.

Reply:
Done